scripts/generate_records/generate_db_executions.js

```diff
@@ -0,0 +1,217 @@
+//@ts-check
+/* eslint-disable no-await-in-loop */
+/* eslint-disable node/no-extraneous-require */
+const cliProgress = require('cli-progress');
+const { ExecutionPgModel, getKnexClient } = require('@cumulus/db');
+const pMap = require('p-map');
+const { randomInt } = require('crypto');
+const minimist = require('minimist');
+const moment = require('moment');
+const range = require('lodash/range');
+const {
+  loadExecutions,
+  loadCollection,
+} = require('./db_record_loaders');
+process.env.DISABLE_PG_SSL = 'true';
+/**
+ *
+ * @typedef {import('knex').Knex} Knex
+ * @typedef {{
+ *   knex: Knex,
+ *   collectionCumulusId: number,
+ *   executionsPerBatch: number,
+ *   model: ExecutionPgModel,
+ * }} BatchParams
+ */
+
+/**
+ *
+ * @param {BatchParams} params
+ * @returns {Promise<void>}
+ */
+const uploadExecutionsBatch = async (
+  {
+    knex,
+    collectionCumulusId,
+    executionsPerBatch,
+    model,
+  }
+) => {
+  await loadExecutions(
+    knex,
+    collectionCumulusId,
+    executionsPerBatch,
+    model,
+    {
+      updated_at: moment().subtract(randomInt(20), 'days').toDate(),
+      original_payload: Math.random() < 0.4 ? { a: 'b' } : null,
```

Comment:
I wonder if we want to make this configurable. As in, maybe I just want to generate consistent executions and not have the random payloads. For this story it makes a ton of sense but as a more generic tool maybe have it optional.

Reply:
yea the exact proportion seemed like it needed to be arbitrary, this is about 2/3 records will have one or the other. I'm not real sure how to reasonably make this an option, but this is a dev tool so I'm assuming you can tweak it if you need. it's such a small amount of data that it won't really affect you if you don't need it

Reply:
I mean, you could add a flag that either adds this randomly-distributed payload or static payloads for each. 

For future use it might be nice to have realistic-looking payloads rather than { a: 'b' } as well. These types of scripts to generate large amounts of data are something we're needing more and more.

That said, I hear you that this doesn't need to do everything here and we can iterate as needed.